### ISSUE TITLE
Which version does support the Insights add-in?

### ISSUE THREAD

**Issue by JurrenG** [MicrosoftDocs/OfficeDocs-WorkplaceAnalytics#173]
[This page](https://docs.microsoft.com/en-us/workplace-analytics/myanalytics/use/add-in) states the supported Outlook 2016 version is 16.0.7924.1 while [this page](https://support.microsoft.com/en-us/help/4486720/november-3-2020-update-for-outlook-2016-kb4486720) states the latest version of Outlook 2016 (Office 2016 MSI version) is 16.0.5071.1000. 

[This page](https://docs.microsoft.com/en-us/workplace-analytics/myanalytics/overview/plans-environments#outlook-support-and-prerequisites) states Outlook 2016 is supported (and after the March 8 updates even Outlook 2013). 

So now I'm a bit confused: Is the most recent version of Outlook (Office 2016 MSI version) supported or not?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 74c447e0-fae6-a016-c6bf-01438701ce17
* Version Independent ID: b6d82917-aee4-2808-00ce-df6c20b75766
* Content: [Insights Outlook add-in - Workplace Intelligence](https://docs.microsoft.com/en-us/workplace-analytics/myanalytics/use/add-in)
* Content Source: [WorkplaceAnalytics/MyAnalytics/Use/add-in.md](https://github.com/MicrosoftDocs/OfficeDocs-WorkplaceAnalytics/blob/public/WorkplaceAnalytics/MyAnalytics/Use/add-in.md)
* Product: **mya**
* GitHub Login: @paul9955
* Microsoft Alias: **v-pausch**

**Comment by paul9955** [comment]
Hello JurrenG,
Thank you for these observations! I'm looking into this now with the product team, and we will update the documentation accordingly. I'll reply here when we've done so.
Again, thanks for this,
 -- Paul Schafer

**Comment by JurrenG** [comment]
Thanks for your swift reply, Paul! Looking forward to the update.

**Comment by paul9955** [comment]
Hi JurrenG,

I heard back from the product team and have updated the docs accordingly. It turns out that the required version of Outlook actually differs for different elements of MyAnalytics. The following articles give the version and build numbers that are required:

- For the **Outlook insights add-in**, see:  [Why don't I see the Insights add-in?](https://docs.microsoft.com/en-us/workplace-analytics/myanalytics/use/add-in#why-dont-i-see-the-outlook-insights-add-in)
- For **inline suggestions**, see:  [Why don't I see inline suggestions?](https://docs.microsoft.com/en-us/workplace-analytics/myanalytics/use/mya-notifications#why-dont-i-see-any-inline-suggestions)

Please let me know if this doesn't answer your question.  
Thank you, 
 -- Paul

**Comment by JurrenG** [comment]
Hi Paul,

I see you closed the issue over the weekend but to me it's not solved.

For the inline suggestions you need the Insights add-in, and the add-in commands for Outlook which are supported in every Outlook version since the Outlook 2013 March 8 2016 update. So the version requirement for Outlook is the only one that really matters here.

The latest version of Outlook MSI is [16.0.5071.1000](https://support.microsoft.com/en-us/help/4486720/november-3-2020-update-for-outlook-2016-kb4486720), and the latest version of Outlook 2016 C2R (which is using a different versioning system) is [Version 2010 (Build 13328.20356)](https://docs.microsoft.com/en-us/officeupdates/update-history-office-2019). 

[This page](https://docs.microsoft.com/en-us/workplace-analytics/myanalytics/use/add-in) still states I need Outlook desktop 2016 version 16.0.7924.1, but where do I get this version of Outlook?  [This page](https://docs.microsoft.com/en-us/workplace-analytics/myanalytics/use/mya-notifications#why-dont-i-see-any-inline-suggestions) still mentions support for Outlook 2016 C2R version 1808 ànd Outlook MSI version 16.0.10730 or greater.

So in short version management is a bit of a mess and deleting version support info just doesn't help. When testing the Insights add-in several months ago the add-in worked fine in Outlook 2016 MSI, but now we are on the brink of delivering MyAnalytics to 40.000+ clients it seems like the support matrix changed. 

Could you please reinstate the support matrix for the Insights add-in update it with versions that are communicated via [this page](https://docs.microsoft.com/en-us/OfficeUpdates/)?

**Comment by paul9955** [comment]
Hi JurrenG,
Thanks for the new comment and the additional details. 
I've contacted people on the product team again and forwarded your entire question. 
We'll get more exact answers for you. Of course, if this involves updates to the docs, I'll point you to them. 
 -- Paul

**Comment by paul9955** [comment]
Hello JurrenG, 
My contact in the product team would like to speak with you to help you with this issue. If you respond with your contact information, I'll pass it on to him. 
Thank you, 
 -- Paul

**Comment by JurrenG** [comment]
Hi Paul, thanks for getting back to me! Since I'm a bit hesitant to share my contact info on the public internet I've asked my Microsoft contact to connect us via a more private channel, hope to speak to you or the PG soon!

**Comment by paul9955** [comment]
Not a problem at all, Jurren! Someone should be contacting you soon.  -- Paul

**Comment by paul9955** [comment]
Hello again! 
I've been in contact with the product team some more about this. They gave me the correct versions and I updated the docs. Here is the info about inline suggestions:
[Inline suggestions](https://docs.microsoft.com/en-us/workplace-analytics/myanalytics/use/mya-notifications#why-dont-i-see-any-inline-suggestions)
And for good measure, about the add-in: 
[Outlook insights add-in](https://docs.microsoft.com/en-us/workplace-analytics/myanalytics/use/add-in#why-dont-i-see-the-outlook-insights-add-in)
Please let me know if you have any further questions. 
Thank you, 
 -- Paul

**Comment by JurrenG** [comment]
Hi Paul, thanks for connecting me to the PG. We had a meeting about this, and this helped us getting more insight into the version requirements and past development of the Insights add-in. We now know what to expect and have a clear plan on how to move forward. Thanks again!

**Comment by paul9955** [comment]
That's great to hear. Thanks for letting me know!

**Comment by Stephapa** [comment]
Hello, 
My customer has a mix of Office 2013 and Office 365 Pro Plus. Since a couple of weeks users with Outlook 2013 complain that Insights addin doesn’t work anymore. They receive a message that asks to upgrade Outlook.
This page mentions a minimum version of  Outlook version 1705 (build 8201.2102) or higher for Windows. Is it something that changed recently ? Is there anyway to make the addin work with Office 2013 ?

**Comment by paul9955** [comment]
Hello Stephane,
I've contacted the product team and we're looking into this for you. I'll let you know as soon as I have an answer.
- Paul

**Comment by Stephapa** [comment]
> Hello Stephane,
> I've contacted the product team and we're looking into this for you. I'll let you know as soon as I have an answer.
> 
> * Paul

Hello Paul, thank you for your answer.

**Comment by paul9955** [comment]
Hi Stephane, I didn't hear back from the product team, so I've contacted them again with your question. Sorry about the delay.

**Comment by paul9955** [comment]
Continuing this discussion in email, so I'll close this thread for now.
